filechooser-portal/Main.vala

```diff
@@ -161,7 +161,6 @@ public class Files.FileChooserPortal : Object {
         });
 
         dialogs[parent_window] = dialog;
-        // dialog.show_all ();
```

Comment:
be careful with `show_all` called on a window or dialog. This should be `present` instead